hawk/app/controllers/application_controller.rb

```diff
@@ -176,7 +176,13 @@ def current_user
   end
 
   def is_god?
-    current_user == "hacluster" || current_user == "root"
+    current_user == "hacluster" || current_user == is_root?
+  end
+
+  def is_root?
```

Comment:
Add `return true if current_user == "root"` please. It's unlikely we'll have anyone using that for login, but just to keep things working the same.

Reply:
done